testData/testConfig.json

```diff
@@ -2,6 +2,12 @@
   "user-namespace": "jupyter-pods",
   "sub-dir": "/lw-workspace",
   "user-volume-size": "10Gi",
+  "gen3-license-user-maps-dynamodb-table": "gen3-license-user-maps",
+  "gen3-license-type": "TEST-LICENSE",
+  "gen3-g3auto-name": "test-license-g3auto",
+  "gen3-max-license-ids": 6,
+  "gen3-g3auto-key": "test_license.txt",
+  "gen3-license-file-path": "stata.lic",
```

Comment:
i did not take the time to read through the feature so i might be wrong, but wouldn't we want to be able to configure the license at the container level (like the "nextflow" config), instead of at the hatchery service level?